app/server/db/requests.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+import PouchDB from 'pouchdb';
+
+export default class DBConnection {
```

Comment:
some notes about PochDB: move this code to shared folder. use global constants like __CLIENT__ or __SERVER__ for making different ways for data fetching